views/block.tmpl

```diff
@@ -55,6 +55,18 @@
 				<div class="fs13 text-secondary pb-1">Block Hash</div>
 				<div class="d-inline-block fs14 break-word rounded font-weight-bold">{{.Hash}}</div>
 			</div>
+			{{range $.AltBlocks}}
+			<div class="text-left lh1rem pt-2 pb-1">
+				<div class="fs13 text-secondary pb-1">
+					Alt. Block (on {{if .IsMainchain}}main chain{{else}}side chain{{end}})
+				</div>
+				<div class="d-inline-block fs14 break-word rounded font-weight-bold">
+					<a href="/block/{{.Hash}}" title="Valid: {{.IsValid}}">
+						{{.Hash}}
+					</a>
+				</div>
+			</div>
+			{{end}}
```

Comment:
This obviously makes the "Block #" parent div taller, but I'm fine with that since this is rare.  What would be bad is if there are several alt blocks, which happens occasionally on testnet.

Maybe we could make it like this to be more compact in the case of more than one alt block:

![image](https://user-images.githubusercontent.com/9373513/80816583-6343d800-8b95-11ea-9c78-3b460a699f7e.png)